Example/PinLayoutSample/UI/Examples/AdjustToContainer/Subviews/ChoiceSelectorView.swift

```diff
@@ -46,21 +46,13 @@ class ChoiceSelectorView: UIView {
 
     override func layoutSubviews() {
         super.layoutSubviews()
-        _ = layout()
+        layout()
```

Comment:
Until the feature is experimental, I would prefer to keep samples as they are for now. But it would be nice to have a more complex sample using autoSizing.

Reply:
Totally agree, I would still update it once the feature is out of experimental just to give a good exemple that do not mutate the frames out of a `layoutSubviews()`, what do you think?

Reply:
I agree, but for the time we keep this feature as experimental it would be nice to have another example that uses that feature in a more complex example.

Reply:
I reverted the sample code modifications and will work on a new sample when the feature go out of experimental mode.